### PR TITLE
fixed filter names

### PR DESCRIPTION
**Pull request by debris**
Also made `CallResult` field public, cause it is used by many public functions

### CODE REVIEW

File: src/lib.rs [hunk 1]
```diff
@@ -28,6 +28,7 @@ pub mod transports;
 pub mod types;
 
 pub use api::{Web3Main as Web3, ErasedWeb3};
+pub use helpers::CallResult;
```
Comment (tomusdrw) [inline src/lib.rs]: helpers are now public, this is unnecessary.

Comment (debris) [inline src/lib.rs]: imo it helps with docs readability -> https://tomusdrw.github.io/rust-web3/web3/api/eth/struct.Eth.html

All functions return `CallResult` and to know what this structure is, I need to open the source code

Comment (tomusdrw) [inline src/lib.rs]: Ah, sorry didn't notice that my PR is not merged yet.
This #33 exposes the whole `helpers` module, so docs will be fixed for `CallResult` as well. That's why I think that re-export is not necessary.

Comment (rphmeier) [inline src/lib.rs]: CallResult also isn't the best name...it should be called "LazyDecode" or something like that

Comment (rphmeier) [inline src/lib.rs]: (and `CallResult::new` everywhere isn't great either, maybe a `lazy_decode` helper function as a constructor would improve readability)

Comment (debris) [inline src/lib.rs]: @rphmeier I agree #38

**Pull request merged**